demo/kubernetes/README.md

```diff
@@ -14,12 +14,13 @@ Note that we are striving for even more smooth integration with Kubernetes in th
 
 ### Deploy Rook
 
-With your Kubernetes cluster running, Rook can be setup and deployed by simply deploying the [rook-operator](rook-operator.yaml) deployment manifest.
+With your Kubernetes cluster running, Rook can be setup and deployed by simply creating the [rook-operator](rook-operator.yaml) deployment and creating an [instance of rook](rook-operator-instance.yaml).
 
```

Comment:
+1. The term Rook-operator-instance seems to implied that they are many operators